Gordon360/Models/CCT/Housing/RA_Status.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[PrimaryKey("Ra_ID", "Status_name")]
+[Table("RA_Status", Schema = "Housing")]
+public partial class RA_Status
```

Comment:
As with the on call tables, it seems like you'll save time and complexity by having one table. You can very easily calculate an RA's current status by searching the status history table (something databases are VERY good at, assuming you add a proper index).